Ginger/Ginger/GeneralWindows/Run Set Pages/NewRunSetPage.xaml

```diff
@@ -41,19 +41,20 @@
                     <RowDefinition Height="60*"/>
                 </Grid.RowDefinitions>
                 <Grid.ColumnDefinitions>
-                    <ColumnDefinition Width="200*"></ColumnDefinition>
-                    <ColumnDefinition Width="270"></ColumnDefinition>
+                    <ColumnDefinition Width="8*"/>
+                    <ColumnDefinition Width="23*"/>
+                    <ColumnDefinition Width="420"></ColumnDefinition>
                 </Grid.ColumnDefinitions>
-                <StackPanel Grid.Row="0" Grid.Column="0" Orientation="Horizontal" HorizontalAlignment="Left" VerticalAlignment="Center" Margin="0,10,0,0" >
+                <StackPanel Grid.Row="0" Grid.Column="0" Orientation="Horizontal" HorizontalAlignment="Left" VerticalAlignment="Center" Margin="0,15,0,5" Height="40" Width="400" >
```

Comment:
specifiing specific width is not seems like correct solution, width should be dynamic based on screen size so for example if big screen been used it should allow showing RunSet name which is longer than 247